### PR TITLE
Remove obsolete remnants of rcm (resource managed) and buildinfo

### PR DESCRIPTION
**Pull request by pshipton**
Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

File: runtime/oti/j9consts.h [hunk 1]
```diff
@@ -610,7 +610,6 @@ extern "C" {
 #define J9_JCL_FLAG_REFERENCE_OBJECTS 0x1
 #define J9_JCL_FLAG_FINALIZATION 0x2
 #define J9_JCL_FLAG_THREADGROUPS 0x8
-#define J9_JCL_FLAG_RESOURCE_MANAGED 0x10
```
[the remark this answers is not in the document]
Comment (pshipton) [reply runtime/oti/j9consts.h]: I'm not sure what the benefit is. I know there are some places that do this, but you can also look at which bits are used to find gaps. I could leave it as a comment.

`/* #define J9_JCL_FLAG_UNUSED_10 0x10 */`

[the remark this answers is not in the document]
Comment (pshipton) [reply runtime/oti/j9consts.h]: Updated

**Comment by keithc-ca** [comment]
jenkins test sanity

**Comment by keithc-ca** [comment]
Let's try plinux again and hope that we won't see https://github.com/eclipse/openj9/issues/968 this time.

jenkins test sanity plinux

**Pull request merged**